### PR TITLE
Re-fetch recommendations when cart updates

### PR DESCRIPTION
**Pull request by Carsten89**
# Pull Request Template

## Description

Re-fetch recommendations when cart updates

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: frontend/config.json [hunk 1]
```diff
@@ -10,12 +10,14 @@
     "h2": "",
     "h3": "",
     "background": "",
-    "textColor": ""
```
Comment (aaron-martin) [inline frontend/config.json:13]: Why is the config.json file committed to the repository? I think this file should be ignored by git.

File: frontend/reducers/index.js [hunk 2]
```diff
@@ -54,6 +55,11 @@ const recommendationsByType = (
         isFetching: false,
         expires: 0,
       });
+    case CLEAR_RECOMMENDATIONS:
```
Comment (aaron-martin) [inline frontend/reducers/index.js:54]: This case looks exactly like the ERROR_RECOMMENDATIONS case so you could use a fall through for this.
```
case ERROR_RECOMMENDATIONS:
case CLEAR_RECOMMENDATIONS:
      return wrapData(state, payload, {
        isFetching: false,
        expires: 0,
      });
```

**Review by aaron-martin — CHANGES_REQUESTED** [review]

File: frontend/config.json (hunk 1 above)
Comment (Carsten89) [reply frontend/config.json:13]: i think this file was added to git before someone put it to the git ignore. will delete it

File: frontend/reducers/index.js (hunk 2 above)
Comment (Carsten89) [reply frontend/reducers/index.js:54]: done

**Review by aaron-martin — APPROVED** [review]

**Pull request merged**